genie-common/src/test/java/com/netflix/genie/common/util/GenieDateFormatUnitTests.java

```diff
@@ -46,6 +46,6 @@ public void canFormat() {
         final StringBuffer buffer = new StringBuffer();
         final FieldPosition pos = Mockito.mock(FieldPosition.class);
 
-        Assert.assertThat(format.format(zero, buffer, pos).toString(), Matchers.is("1970-01-01T00:00:00.000Z"));
+        Assert.assertThat(format.format(zero, buffer, pos).toString(), Matchers.is("1970-01-01T00:00:00.000+0000"));
```

Comment:
I'm pretty sure this is a breaking change. Certain libraries like Kragle and I believe our UI expect the dates in the previous format with the `Z` at the end.